recipes/svgwrite/0.1.0/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class SvgwriteConan(ConanFile):
+    name = "svgwrite"
+    version = "0.1"
```

Comment:
Should this be 0.1.0?

Reply:
fixed! The control performed by the bot,  which found the inconsistency between the versions, could it be added to the hooks?